sdk/src/sharedTest/java/io/customer/sdk/CustomerIOTest.kt

```diff
@@ -103,6 +103,17 @@ class CustomerIOTest : BaseTest() {
         updatedConfig.trackingApiHostname shouldBeEqualTo givenTrackingApiUrl
     }
 
+    @Test
+    fun deviceToken_givenGetValue_expectDeviceRepositoryGetDeviceToken() {
```

Comment:
Could we add a test when there hasn't been been a device token set yet? Where you expect to get `null`? 

Also, a non-blocking idea to share, I think that this feature benefit better from integration tests. 

```kotlin
assertNull(CustomerIO.instance().deviceToken)

CustomerIO.registerDeviceToken(givenToken)

assertEqual(CustomerIO.instance().deviceToken, givenToken)
```

This feels closer to the way customers will use it so I think it's a higher quality test.